src/components/articles/createArticles/createArticles.js

```diff
@@ -137,6 +142,15 @@ export class CreateArticles extends React.Component {
           theme="snow"
           required
         />
+        <div className="form-group">
+          <label>taglist</label>
+          <input
+            type="text"
+            className="form-control form-control-md"
+            onChange={(ev) => this.handleChangeField('tags', ev)}
+            value={tags}
```

Comment:
Add some instructions to tell the user that tags are separated by commas.

Reply:
I have added them

Reply:
You should also either put it below the input or together with the label but in brackets because when the user begins to type, the placeholder disappears.